ui/imports/shared/popups/SettingsDirtyToastMessage.qml

```diff
@@ -24,6 +27,14 @@ Rectangle {
         saveChangesButton.forceActiveFocus()
     }
 
+    QtObject {
+        id: d
+
+        readonly property string defaultSaveChangesText: qsTr("Save changes")
```

Comment:
... and then you wouldn't need these defaults

Reply:
Thank you for your suggestion! However, I'm confused on that, since if the properties are an alias, each consumer of the component will need to initialize the values and what I wanted here is for the component itself to have default values, since only some consumers will need to specify different values. By default, consumers do not need to initialize them.

Reply:
You can put the defaults where the alias points to :)

Reply:
Ah, that's true! Done! I like more your solution! :) Can you check it again?!

Reply:
Yeah, nice! Much simpler right? :)